ckanext/switzerland/templates/showcase/snippets/showcase_info.html

```diff
@@ -20,6 +20,17 @@ <h1 class="heading">{{ pkg.title or pkg.name }}</h1>
             {% if pkg.author %}
                 <span class="smallest">{{_('Submitted by')}}</span>
                 <p>{{ pkg.author }}</p>
+                {% if pkg.author_twitter %}
+                    <a href="{{ pkg.author_twitter}}" target="_blank">
```

Comment:
```suggestion
                    <a href="{{ pkg.author_twitter }}" target="_blank">
```

Reply:
fix in commit [884fbf8](https://github.com/opendata-swiss/ckanext-switzerland-ng/pull/223/commits/884fbf8a08fa28bfcfc66ca33d08f56e47ec2a06)